src/platforms/python/common/configuration/integrations/loguru.mdx

```diff
@@ -51,50 +46,70 @@ logger.exception("An exception happened")
 
 - There will be an error event with the message `"I am an event"`.
 - `"I am a breadcrumb"` will be attached as a breadcrumb to that event.
-- `bar` will end up in the event's `extra` attributes.
-- `"An exception happened"` will send the current exception from `sys.exc_info()` with the stack trace and everything to the Sentry Python SDK. If there's no exception, the current stack will be attached.
-- The debug message `"I am ignored"` will not surface anywhere. To capture it, you need to lower `level` to `DEBUG` in `LoguruIntegration` args.
+- `bar` will end up in the `extra` attributes of that event.
+- `"An exception happened"` will send the current exception from `sys.exc_info()` with the stack trace to the Sentry. If there's no exception, the current stack will be attached.
+- The debug message `"I am ignored"` will not be captured by Sentry. To capture it, you need to lower `level` to `DEBUG` in `LoguruIntegration`.
 
-## Ignoring a logger
+### Ignoring a logger
 
-Sometimes a logger is extremely noisy and spams you with pointless errors. You can completely ignore that logger by calling `ignore_logger`:
+Sometimes a logger can be noisy. You can ignore that logger by calling `ignore_logger`:
 
 ```python
 # As most of the logic is proxied to `logging` integration, we use it instead of `loguru` integration.
 from sentry_sdk.integrations.logging import ignore_logger
 
-
 ignore_logger("a.spammy.logger")
 
-# this will work with `logging`'s logger too
-logger = logging.getLogger("a.spammy.logger")
-logger.error("hi")  # no error sent to sentry
```

Comment:
Why did you remove these strings?

Reply:
I checked, they are in the final version.